apps/condo/migrations/20240902220300-0418_ticket_ticket_unitname_ticket_ticket_ispayable_and_more.js

```diff
@@ -0,0 +1,74 @@
+
+exports.up = async (knex) => {
+    await knex.raw(`
+    BEGIN;
+--
+-- Create index ticket_unitName on field(s) unitName of model ticket
+--
+CREATE INDEX "ticket_unitName" ON "Ticket" USING btree ("unitName");
+--
+-- Create index ticket_isPayable on field(s) isPayable of model ticket
+--
+CREATE INDEX "ticket_isPayable" ON "Ticket" USING btree ("isPayable");
+--
+-- Create index ticket_isEmergency on field(s) isEmergency of model ticket
+--
+CREATE INDEX "ticket_isEmergency" ON "Ticket" USING btree ("isEmergency");
+--
+-- Create index ticket_isWarranty on field(s) isWarranty of model ticket
+--
+CREATE INDEX "ticket_isWarranty" ON "Ticket" USING btree ("isWarranty");
+--
+-- Create index ticket_statusReopenedCounter on field(s) statusReopenedCounter of model ticket
+--
+CREATE INDEX "ticket_statusReopenedCounter" ON "Ticket" USING btree ("statusReopenedCounter");
+--
+-- Create index ticket_assignee_exec_org on field(s) assignee, executor, organization of model ticket
+--
+CREATE INDEX "ticket_assignee_exec_org" ON "Ticket" USING btree ("assignee", "executor", "organization");
+--
+-- Create index ticket_organization_status on field(s) organization, status of model ticket
+--
+CREATE INDEX "ticket_organization_status" ON "Ticket" USING btree ("organization", "status");
```

Comment:
You need to manually add condition ("deletedAt" IS null).
Example in `20240814005507-0417_billingreceipt_billingreceipt_period_context.js`

Reply:
thank you, added it